test/CMakeLists.txt

```diff
@@ -528,7 +528,7 @@ function(add_custom_test NAME)
     set(is_gfx90a_check)
     set(GFX90A_TEST_DEFAULT FALSE)
     option_support_check(${PARSE_GFX90A_ENABLED} ${PARSE_GFX90A_DISABLED} ${GFX90A_TEST_DEFAULT} is_gfx90a_check)
-    bool_and_f(${MIOPEN_TEST_GFX90A} ${is_gfx90A_check} is_gfx90a_check)
+    bool_and_f(${MIOPEN_TEST_GFX90A} ${is_gfx90a_check} is_gfx90a_check)
```

Comment:
This line is causing some issues in CI.

Could we consolidate all "MIOPEN_TEST_GFX90A" to "MIOPEN_TEST_GFX90a", or change existing "MIOPEN_TEST_GFX90a" in develop line to "MIOPEN_TEST_GFX90A"? :)

Reply:
Yes. I think use lowercase to represent ```GFX90a``` is better. 
Modified.